File: solo/easy/01.onboarding.php
<?
while (TRUE){
    fscanf(STDIN, "%s",$e);
    fscanf(STDIN, "%d",$d);
    fscanf(STDIN, "%s",$E);
    fscanf(STDIN, "%d",$D);
    echo((($d<$D)?$e:$E)."\n");
}
?>

File: solo/easy/01.onboarding.ml
while true do
    let e=input_line stdin in 
    let d=int_of_string (input_line stdin) in 
    let f=input_line stdin in 
    let c=int_of_string (input_line stdin) in 
    if d<c then print_endline e else print_endline f;
done;